index.js

```diff
@@ -197,13 +197,19 @@ function init (ssb, config) {
 
         state.newAuthorListeners.forEach(fn => fn({ groupId: data.groupId, newAuthors: [ssb.id] }))
 
-        if (!opts.addPOBox) return cb(null, data)
-        else {
-          scuttle.group.addPOBox(data.groupId, (err, poBoxId) => {
-            if (err) cb(err)
-            cb(null, { ...data, poBoxId })
-          })
-        }
+        // addMember the admin
+        scuttle.group.addMember(data.groupId, [ssb.id], {}, (err) => {
```

Comment:
hey @arj03 I've changed something about groups - this changes the group creation flow to:
- mint group keys
- publish `group/init` + persist keys
- :new: publish `group/add-member` explicitly adding admin to group (this to make backup recovery easier)
- (optionally) mint po box for group

Reply:
@mixmix I was thinking what happens if you crash between init and addMember, but I see you have a "add a migration" in ahau that takes care of this. I'm assuming it will be run on every start? So all good :-)